trl/trainer/grpo_trainer.py

```diff
@@ -407,7 +410,7 @@ def __init__(
         if self.beta == 0.0:
             # If beta is 0.0, the reference model is not needed
             self.ref_model = None
-        elif is_deepspeed_zero3_enabled():
+        elif is_deepspeed_zero3_enabled() or args.fsdp_config is not None:
```

Comment:
Good catch @LeonEricsson ! Since we inherit from `transformers.Trainer` I believe this should work:
```suggestion
        elif is_deepspeed_zero3_enabled() or self.is_fsdp_enabled is not None:
```

Reply:
but `Trainer` isn't initialized until [later](https://github.com/huggingface/trl/blob/9fbf125451e8e45547f4c4174573e16f0e35354d/trl/trainer/grpo_trainer.py#L551)

Reply:
yes i am getting the same error too